### PR TITLE
Fix how Bootstrap-Notfy is used by Pi-hole

### PR DESCRIPTION
**Pull request by DL6ER**
# What does this implement/fix?

See title, this follows the documentation at https://grotesquegentleadvance--samkhaled.repl.co/

**Related issue or feature (if applicable):** N/A

**Pull request in [docs](https://github.com/pi-hole/docs) with documentation (if applicable):** N/A

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

## Checklist:

- [x] The code change is tested and works locally.
- [x] I based my code and PRs against the repositories `developmental` branch.
- [x] I [signed off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits. Pi-hole enforces the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions
- [x] I [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) all my commits. Pi-hole requires signatures to verify authorship
- [x] I have read the above and my PR is ready for review.

### CODE REVIEW

File: scripts/pi-hole/js/utils.js [hunk 1]
```diff
@@ -83,63 +83,51 @@ function padNumber(num) {
   return ("00" + num).substr(-2, 2);
 }
 
-var info = null;
+var showAlertBox = null;
 function showAlert(type, icon, title, message) {
-  var opts = {};
-  title = "&nbsp;<strong>" + title + "</strong><br>";
+  const options = {
```
Comment (DL6ER) [inline scripts/pi-hole/js/utils.js:88]: We use `const` here. I will details the *why* below in case anyone doesn't know (I will admit: I didn't know).

In JavaScript, `const` is used to declare a variable that cannot be reassigned. However, it does not make the variable immutable. This means that while you cannot reassign a `const` variable to a new value, you can still modify the properties of an object that is assigned to a `const` variable.

Here, the `options` and `settings` objects are declared using `const`. This means that you cannot reassign them to a new value. However, the properties of these objects can still be modified.

For example, the `options` and `settings` objects are modified based on the `type` parameter. The `icon` property of the `options` object is set to a default value if it is not provided, and the `delay` property of the `settings` object is multiplied by 2 if the `type` is "warning" or "error".

In summary, `const` in JavaScript does not make objects immutable, it only prevents the variable from being reassigned to a new value which seems a bit counter-intuitive given the name "const".

File: queries.lp [hunk 2]
```diff
@@ -194,7 +194,6 @@ mg.include('scripts/pi-hole/lua/header_authenticated.lp','r')
 <script src="<?=pihole.fileversion('scripts/vendor/bootstrap-select.min.js')?>"></script>
 <script src="<?=pihole.fileversion('scripts/pi-hole/js/ip-address-sorting.js')?>"></script>
 <script src="<?=pihole.fileversion('scripts/vendor/daterangepicker.min.js')?>"></script>
-<script src="<?=pihole.fileversion('scripts/pi-hole/js/utils.js')?>"></script>
```
Comment (DL6ER) [inline queries.lp:197]: `utils.js` is already included on all pages from the header. No need to do it a second time here.

**Review by PromoFaux — APPROVED** [review]
Tested OK

**Comment by yubiuser** [comment]
I'm not sure if this PR broke it, but I thought we had some kind of fade in/out animation for the notifications. But they don't fade anymore...

____

**Edit**
 Was broken before. Fixed by https://github.com/pi-hole/web/pull/2802
